File: Agnello backend/Agnello.Persistence/Domain/Produto.cs
namespace Agnello.Persistence.Domain;

public class Produto
{
    public int Id { get; set; }
    public string Nome { get; set; } = string.Empty;
    public string? Tipo { get; set; }
    public string? Regiao { get; set; }
    public string? Uva { get; set; }
    public decimal? TeorAlcoolico { get; set; }
    public int? Intensidade { get; set; }
    public int? Encorpado { get; set; }
    public int? Acucar { get; set; }
    public int? Acidez { get; set; }
    public decimal Valor { get; set; }
    public int QuantidadeEstoque { get; set; }
    public decimal? AvaliacaoCritica { get; set; }
}

File: Agnello frontend/app/src/main/java/br/com/fiap/agnellofrontend/data/model/Produto.kt
package br.com.fiap.agnellofrontend.data.model

data class Produto(
    val id: Int = 0,
    val nome: String,
    val tipo: String? = null,
    val regiao: String? = null,
    val uva: String? = null,
    val teorAlcoolico: Double? = null,
    val intensidade: Int? = null,
    val encorpado: Int? = null,
    val acucar: Int? = null,
    val acidez: Int? = null,
    val valor: Double,
    val quantidadeEstoque: Int,
    val avaliacaoCritica: Double? = null
)
